src/worker.py

```
@@ -1095,6 +1095,27 @@ async def api_join(req, env):
+        if act_detail and act_detail.host_id != user["id"]:
+            joiner = decrypt(act_detail.joiner_name or "", env.ENCRYPTION_KEY)
+            activity_title = act_detail.title or "your activity"
+            await _create_notification(
+                env,
+                user_id=act_detail.host_id,
+                type_="new_enrollment",
+                title="New participant joined",
+                message=f"{joiner} joined {activity_title}",
+                related_id=act_id,
+            )
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)